### ISSUE TITLE
Fails to start with long username

### ISSUE THREAD

**Issue by alexandrosm77** [rancher-sandbox/rancher-desktop#1769]
### Actual Behavior

Error log after a clean installation on a M1 apple with a long username.
Rancher desktop does not start

### Steps to Reproduce

Install rancher-desktop on M1 with username longer than 19 chars (alexandros.mystridis is 20 chars)

### Result

`2022-03-09T11:28:47.334Z: Running command limactl list --json...
2022-03-09T11:28:47.424Z: Running command limactl start --tty=false /Users/alexandros.mystridis/Library/Application Support/rancher-desktop/lima/_config/0.yaml...
time="2022-03-09T11:28:47Z" level=fatal msg="instance name \"0\" too long: \"/Users/alexandros.mystridis/Library/Application Support/rancher-desktop/lima/0/ssh.sock.1234567890123456\" must be less than UNIX_PATH_MAX=104 characters, but is 104"
2022-03-09T11:28:47.516Z: + limactl start --tty=false /Users/alexandros.mystridis/Library/Application Support/rancher-desktop/lima/_config/0.yaml
2022-03-09T11:28:47.517Z: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
2022-03-09T11:28:47.518Z: Error starting lima: Error: /Applications/Rancher Desktop.app/Contents/Resources/resources/darwin/lima/bin/limactl exited with code 1
    at ChildProcess.<anonymous> (/Applications/Rancher Desktop.app/Contents/Resources/app.asar/dist/app/background.js:1:8692)
    at ChildProcess.emit (node:events:394:28)
    at Process.ChildProcess._handle.onexit (node:internal/child_process:290:12)`

### Expected Behavior

Rancher starts normally

### Rancher Desktop Version

1.1.1

### Rancher Desktop K8s Version

v1.22.5

### Which container runtime are you using?

moby (docker cli)

### What operating system are you using?

macOS

### Operating System / Build Version

macOS Monterey 12.1

### What CPU architecture are you using?

arm64 (Apple Silicon)

**Comment by asantos2000** [comment]
The same for Mac Os  Monterey (12.01), Intel processor.

**Comment by jandubois** [comment]
> Probably a duplicate of #797

Yes, it is a duplicate, please use workaround from https://github.com/rancher-sandbox/rancher-desktop/issues/797#issuecomment-996510877 until this fixed.
